WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/PrivacyRing/PrivacyRingViewModel.cs

```diff
@@ -183,4 +163,23 @@ private IEnumerable<PrivacyRingItemViewModel> CreatePocketSegments(IEnumerable<P
 			start = end;
 		}
 	}
+
+	private void SetMargins()
+	{
+		Margin = new Thickness(Width / 2, Height / 2, 0, 0);
+		NegativeMargin = new Thickness(Margin.Left * -1, Margin.Top * -1, 0, 0);
```

Comment:
Nit, ignore if you want.

```suggestion
		NegativeMargin = Margin * -1;
```